modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMarketing.macro

```diff
@@ -5,4 +5,89 @@ definition {
 		Click(locator1 = "PRMMarketing#APPLY_FOR_FUNDS");
 	}
 
+	@summary = "Verify the activity fills made by the user"
+	macro viewMDFActivity {
+
+		Click(
+			key_textButton = "${activityName}",
+			locator1 = "PRMMDFRequest#TEXT_BUTTON");
+
+		AssertElementPresent(
+			key_fieldName= "Activity name",
+			key_fieldValue = "${activityName}",
+			locator1 = "PRMMDFRequest#SUMMARY_FIELD");
+
+		AssertElementPresent(
+			key_fieldName= "Type of Activity",
+			key_fieldValue = "${typeOfActivity}",
+			locator1 = "PRMMDFRequest#SUMMARY_FIELD");
+
+		AssertElementPresent(
+			key_fieldName= "Tactic",
+			key_fieldValue = "${tactic}",
+			locator1 = "PRMMDFRequest#SUMMARY_FIELD");
+
+		AssertElementPresent(
+			key_fieldName= "Marketing activity",
+			key_fieldValue = "${activityDescription}",
+			locator1 = "PRMMDFRequest#SUMMARY_FIELD");
+
+		AssertElementPresent(
+			key_fieldName= "Start Date",
+			key_fieldValue = "${startDate}",
+			locator1 = "PRMMDFRequest#SUMMARY_FIELD");
+
+		AssertElementPresent(
+			key_fieldName= "End Date",
+			key_fieldValue = "${endDate}",
+			locator1 = "PRMMDFRequest#SUMMARY_FIELD");
+
+		AssertElementPresent(
+			key_fieldName= "${expense}",
+			key_fieldValue = "${budget}",
+			locator1 = "PRMMDFRequest#SUMMARY_FIELD");
+
+		AssertElementPresent(
+			key_fieldName= "Is a lead list an outcome of this activity?",
+			key_fieldValue = "${leadList}",
+			locator1 = "PRMMDFRequest#SUMMARY_FIELD");
+	}
+	
+	@summary = "Verify the fills done by the user on the Campaign Information"
```

Comment:
Hi @marinaasant0s, could you please update this summary to: "View the goals information on MDF Review Page